modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/package.scala

```diff
@@ -41,58 +41,48 @@ package object engine {
     * Changes the `Status` and returns the new `QState`.
     *
     * It also takes care of initiating the execution when transitioning to
-    * `Running` status.
+    * `Running` `Status`.
     */
   def switch(q: EventQueue)(st: Status): Engine[QState] =
     modify(QState.status.set(_, st)) *>
-    whenM (st == Status.Running) (prime *> execute(q)) *>
+    // TODO: Make Status an Equal instance
+    whenM (st == Status.Running) (execute(q)) *>
```

Comment:
Actually, I wanted to get rid of all those `get`s in all the functions of that file. They all end in `*> get` so there should be a way to factor that out. I'll try to take of care of it now for this review.